src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -698,8 +692,8 @@ class MatrixDelayedUpdateCUDA
     auto& queue                = engine_leader.cuda_handles_.getResource().queue;
     auto& h_cublas             = engine_leader.cuda_handles_.getResource().h_cublas;
     auto& updateInv_buffer_H2D = engine_leader.mw_mem_handle_.getResource().updateInv_buffer_H2D;
-    const int norb             = engine_leader.get_psiMinv().rows();
-    const int lda              = engine_leader.get_psiMinv().cols();
+    const int norb             = engine_leader.invRow.size();
```

Comment:
if instead this is a indirect consistency check, an assert would probably be a better way to handle it.  If there is a chance they may not be consistent some documentation somewhere would be useful.